### PR TITLE
Patch some bugs in MCE field

### PR DESCRIPTION
**Pull request by jcowher**

### CODE REVIEW

File: wp-forms-api/class-wp-forms-api.php [hunk 1]
```diff
@@ -505,13 +506,26 @@ static function render_element( $element, &$values, $form = null ) {
 				break;
 
 			case 'mce':
+				if ( ! user_can_richedit() ) {
```
Comment (bendoh) [inline wp-forms-api/class-wp-forms-api.php]: Can we do `if(` without the space so it's at least internally consistent throughout this file?

File: wp-forms-api/class-wp-forms-api.php [hunk 2]
```diff
@@ -505,13 +506,26 @@ static function render_element( $element, &$values, $form = null ) {
...
+					// User doesn't have capabilities to richedit - just display
+					// a regular textarea with html tags allowed
+					$element['#type'] = 'textarea';
```
Comment (bendoh) [inline wp-forms-api/class-wp-forms-api.php:512]: Oh, duh! Wooooops. Nice cleanup.

File: wp-forms-api/class-wp-forms-api.php [hunk 3]
```diff
@@ -505,13 +506,26 @@ static function render_element( $element, &$values, $form = null ) {
...
+					$element['#allow_html'] = true;
+
+					if ( ! isset( $attrs['rows'] ) ) {
```
Comment (bendoh) [inline wp-forms-api/class-wp-forms-api.php]: Same here, and no space after unary operators like `!`

File: wp-forms-api/class-wp-forms-api.php [hunk 4]
```diff
@@ -755,8 +769,12 @@ static function process_element( $element, &$values, &$input ) {
 		else if( isset( $input[$element['#key']] ) ) {
 			$element['#value'] = $input[$element['#key']];
 
+			// Sanitization of fields that allow html
```
Comment (bendoh) [inline wp-forms-api/class-wp-forms-api.php:772]: Again, the spacing after `if` should match what's already in the file

**Comment by jcowher** [comment]
Thanks @bendoh - just pushed another commit with these changes

**Comment by bendoh** [comment]
Magnifique! 👯

**Pull request merged**